spec/models/event_spec.rb

```diff
@@ -117,6 +117,103 @@ def emails_sent
     end
   end
 
+  describe 'audience_mentioned' do
+    let(:discussion_audience){ Audience.discussion.name }
+    let(:group_audience){ Audience.group.name }
+
+    it 'notifies group members' do
+      comment = create :comment, parent: parent_comment, discussion: discussion, body: "hey @#{group_audience} good day"
```

Comment:
also, I recommend testing this explicitly here. `hey @group good day` rather than interpolating it.